src/api/airdropEligibility.js

```diff
@@ -0,0 +1,11 @@
+const { get } = require('request-promise')
+
+module.exports = async (dvf, ethAddress) => {
+  if (ethAddress) {
+    const response = await get(
+      `${dvf.config.api}/v1/trading/r/airdropEligibility?ethAddress=${ethAddress}`
```

Comment:
You can also use `get-generic` like in https://github.com/DeversiFi/dvf-client-js/blob/master/src/api/getPublicPermissions.js